samples/app-identity-linking-with-sso/nodejs/README.md

```diff
@@ -39,12 +45,14 @@ Messaging Extention mapping
 
   ![ME search](Images/MESearch.png)
 
-  ![ME search](Images/MESearchLinkCard.jpg)
+  ![ME search](Images/MESearchLinkCard.png)
    
 - ME Link unfurling
 
   ![ME Link unfurling](Images/MELinkUnfurlingCard.png)
 
+- On Click of `Disconnect from facebook` or `Disconnect from google`. User's facebook/google data will be removed from the data base.
```

Comment:
Database
Also, are we using any database or just saving it in memory.
So we can add a extensibility note like, you can use any memory like DB or in memory or in-file.